loader.js

```diff
@@ -178,4 +177,14 @@ module.exports = function(source) {
   const allCode = headerCode + os.EOL + importOutputCode + os.EOL + outputCode + os.EOL;
 
   return allCode;
+}
+
+// make graphql transformer available for other bundlers
+exports.transform = transform;
```

Comment:
my understanding of how `exports` and `module.exports` work together is that `module.exports` is going to override anything you put on `exports` [(unless you explicitly make them the same object)](https://medium.freecodecamp.org/node-js-module-exports-vs-exports-ec7e254d63ac).